WordPress/Classes/ViewRelated/Site Creation/FinalAssembly/SiteAssemblyWizardContent.swift

```diff
@@ -62,7 +63,7 @@ final class SiteAssemblyWizardContent: UIViewController {
 
         hidesBottomBarWhenPushed = true
         installButtonViewController()
-        WPAnalytics.track(.enhancedSiteCreationSuccessLoading)
+        SiteCreationAnalyticsHelper.trackSiteCreationSuccessViewed(siteCreator.design)
```

Comment:
Loading and viewed seemed to be flip-flopped before, however, I'm not sure if this was intentional or not. This goes into a step we didn't modify. @antonis do you know if the order of events on Android are:
A)
- `enhanced_site_creation_site_design_preview_viewed` 
- `site_created`
- `enhanced_site_creation_site_design_preview_loading`
- `enhanced_site_creation_site_design_preview_loaded` 

Or are they: 

B)
- `enhanced_site_creation_site_design_preview_loading`
- `site_created`
- `enhanced_site_creation_site_design_preview_viewed` 
- `enhanced_site_creation_site_design_preview_loaded`

Reply:
For just previewing (not the post-site-creation preview, but prior to site creation) the order is: `viewed` -> `loading` -> `loaded`. I'd expect it to be the same upon site creation, but I will double check that to be sure.

Reply:
Just checked it, and WordPress-Android is currently using different events for tracking the preview, and the order is indeed different. I observed this sequence of events after site creation:

`enhanced_site_creation_success_loading`
:arrow_down:
`site_created`
:arrow_down:
`enhanced_site_creation_success_preview_viewed`
:arrow_down:
`enhanced_site_creation_success_preview_loaded`

Neither of those have the `template` property, currently, and I believe these events are the same as emitted in the existing flows.

Reply:
Hey @mkevins 👋 

![Screenshot 2020-11-25 at 3 45 23 PM](https://user-images.githubusercontent.com/304044/100235624-51f95900-2f35-11eb-9840-12be93d6028b.png)

I tried to visualise your observations diffing the log from the two platforms (current Android with the events introduced in iOS with this PR)

> Neither of those have the template property, currently, and I believe these events are the same as emitted in the existing flows.

Yes, I haven't changed this on Android. I still use the `enhanced_site_creation_success_*` events of the existing old flow that do not have carry the template information. I've probably missed this change 😕 

My answer to @chipsnyder above might have been misleading since I was referring to the order of the `enhanced_site_creation_site_design_preview_*` events and not the `enhanced_site_creation_success_*` ones 😞 

I drafted a PR that aligns with this iOS PR here https://github.com/wordpress-mobile/WordPress-Android/pull/13479

I guess we can either merge both (iOS and Android) PRs or keep the code as is and do not track template (the old analytics events are in sync in that case).

Another way would be to add the template in the payload of the old `enhanced_site_creation_success_*` events. This will allow us to differentiate the preview of a design with the preview of a created site.